src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_StrandOddsRatio.java

```diff
@@ -69,4 +70,18 @@
         final double ratio = StrandOddsRatio.calculateSOR(table);
         return Collections.singletonMap(getKeyNames().get(0), StrandOddsRatio.formattedValue(ratio));
     }
+
+    @Override
+    protected Map<Allele,Double> calculateReducedData(AlleleSpecificAnnotationData<List<Integer>> combinedData) {
+        final Map<Allele,Double> annotationMap = new HashMap<>();
+        final Map<Allele, List<Integer>> perAlleleData = combinedData.getAttributeMap();
+        final List<Integer> refStrandCounts = perAlleleData.get(combinedData.getRefAllele());
+        for (final Allele a : perAlleleData.keySet()) {
+            List<Integer> altStrandCounts = perAlleleData.get(a);
+            int[][] refAltTable = new int[][] {new int[]{refStrandCounts.get(0),refStrandCounts.get(1)},new int[]{altStrandCounts.get(0),altStrandCounts.get(1)}};
```

Comment:
You could use the FORWARD and REVERSE here too.

Reply:
done